a2example/vagrantA2/.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
+# with Vagrant so that encoded files can properly run with Vagrant.
+
+if ENV["VAGRANT_INSTALLER_EMBEDDED_DIR"]
+  require File.expand_path(
+    "rgloader/loader", ENV["VAGRANT_INSTALLER_EMBEDDED_DIR"])
+else
+  raise "Encoded files can't be read outside of the Vagrant installer."
```

Comment:
Would be great to add what the user should do next. The passive voice ("files can't be read") left me confused as to what is reading the files and what I need to do to fix the problem. Still, any error message is better than no error message, so thanks!

```suggestion
  raise "The Vagrant installer can't read outside encrypted files"
```